driver/main_args.ml

```diff
@@ -1037,6 +1037,10 @@ module type Ocamldoc_options = sig
   val _vmthread : unit -> unit
 end
 
+module type Ocamltest_options = sig
+  val _color : string -> unit
+end
```

Comment:
I'm not fond of the idea of using the driver/ machinery for *some* of ocamltest arguments, but only a small fraction. Can you achieve the same result by just adding a `-color` flag in ocamltest/options.ml?

Reply:
I didn't do it because it duplicates the docstring of the `-color` flag. This annoyed me, I looked around and found I could avoid the duplication somewhat like `ocamldoc/odoc_args.ml` does:

```ocaml
module Options = Main_args.Make_ocamldoc_options(struct
    include Main_args.Default.Odoc_args
    let _I s = Odoc_global.include_dirs := s :: !Odoc_global.include_dirs
    let _H s = Odoc_global.hidden_include_dirs := s :: !Odoc_global.hidden_include_dirs
    let _impl s = Odoc_global.files := !Odoc_global.files @ [Odoc_global.Impl_file s]
    let _intf s = Odoc_global.files := !Odoc_global.files @ [Odoc_global.Intf_file s]
end)


(** The default option list *)
let default_options = Options.list @
[
```